configuration/security.md

```diff
@@ -73,6 +73,8 @@ This command will delete the `pi` user but will leave the `/home/pi` folder. If
 ```bash
 sudo deluser -remove-home pi
 ```
+This command will result in a warning that the group `pi` has now more members. This group is, however, removed along with the `pi` user itself by the deluser command, so that the warning can be ignored.
```

Comment:
"now more members" -> "no more members" ?
I think the second sentence might be more readable as something like "The `deluser` command removes both the `pi` user and the `pi` group though, so the warning can be safely ignored."

(Note that I'm only editing the text for grammar here - I've not tried this on my Pi so I don't know if it's technically correct!)

Reply:
Correct. Sorry for that.